changelog.d/6217.misc

```diff
@@ -0,0 +1,2 @@
+Remove Auth.check method.
+
```

Comment:
```suggestion
```

Reply:
don't think it makes any difference, but thanks